internal/state/resolver/service_resolver_test.go

```diff
@@ -99,22 +101,25 @@ var _ = Describe("ServiceResolver", func() {
 							Type:   intstr.Int,
 							IntVal: 8080,
 						},
+						Protocol: v1.ProtocolTCP,
 					},
 					{
 						Port: 443,
 						TargetPort: intstr.IntOrString{
 							Type:   intstr.String,
 							StrVal: "8443",
```

Comment:
I think `StrVal: "8443"` can be misleading, because k8s will set the int value if YAML includes "8443", not string.